test/test-functions

```diff
@@ -2463,7 +2418,35 @@ inst_binary() {
             exit 1
         fi
     done < <(LC_ALL=C ldd "$bin" 2>/dev/null)
-    inst_simple "$bin" "$target"
+
+    # Same as above, but we need to wrap certain libraries unconditionally
+    #
+    # tar - called by machinectl in TEST-25
+    if get_bool "$IS_BUILT_WITH_ASAN" && [[ "$bin" =~ /(tar)$ ]]; then
+        wrap_binary=1
+    fi
+
+    if get_bool "$wrap_binary"; then
```

Comment:
> But I can't think of any scenario where it would be useful (at least in current situation)

https://github.com/bus1/dbus-broker/issues/291 (where `dbus-broker` was instrumented along with `systemd`) would be an example of a scenario where it could be useful :-)

Reply:
Good point, updated :-)